app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1> Welcome to Users List</h1>
\ No newline at end of file
+<div class = 'user'>
+<h1> Welcome to Users List</h1>
+<div class = 'user_container'>
+<% @users.each do |user|%>
+<div class = 'user_row'>
+<div class = 'user_image'>
+
+</div>
+<div class = 'user_block'>
+<h3><%= link_to user.name, user_path(user)  %></h3>
+<img src="<%= link_to user.photo %>" >
```

Comment:
- [ ] as it is in the wireframe your image is showing image link instead of the image kindly fix this so it could match the wireframe
![Screenshot from 2023-06-23 11-21-45](https://github.com/imranhossainemi/blog-website/assets/66097371/8918d74a-ea5b-4d3b-b64d-9dbd095780ce)
wireframe :point_down: 
![Screenshot from 2023-05-11 17-30-55](https://github.com/imranhossainemi/blog-website/assets/66097371/c2329ffe-49b1-4d1f-b718-2e6284eee7c4)